prow/plugins/config.go

```diff
@@ -61,6 +61,7 @@ type Configuration struct {
 	Cat                        Cat                    `json:"cat,omitempty"`
 	CherryPickUnapproved       CherryPickUnapproved   `json:"cherry_pick_unapproved,omitempty"`
 	ConfigUpdater              ConfigUpdater          `json:"config_updater,omitempty"`
+	Dcos                       []Dco                  `json:"dco,omitempty"`
```

Comment:
Can we keep this as `Dco` instead of `Dcos` to be consistent with the json `dco`?

Reply:
Fixed